patches/chrome-android-java-src-org-chromium-chrome-browser-util-FeatureUtilities.java.patch

```diff
@@ -10,3 +10,12 @@ index 3b19402872ef636a37019cc6fb95b61607b8e281..f4cf9c43834262269e97dbbe512b4f3c
          if (sIsBottomToolbarEnabled == null) {
              ChromePreferenceManager prefManager = ChromePreferenceManager.getInstance();
  
+@@ -504,7 +505,7 @@ public class FeatureUtilities {
+             ChromePreferenceManager prefManager = ChromePreferenceManager.getInstance();
+ 
+             sIsNightModeAvailable = prefManager.readBoolean(
+-                    ChromePreferenceManager.NIGHT_MODE_AVAILABLE_KEY, false);
++                    ChromePreferenceManager.NIGHT_MODE_AVAILABLE_KEY, true);
```

Comment:
This is part of an upstream commit in C80. https://chromium.googlesource.com/chromium/src/+/32bab2eff398787266e349509843909978978bfc